README.md

```diff
@@ -15,30 +15,26 @@ For further information on this and other VUI components, see the docs at [ui.va
 bower install vui-typography
 ```
 
-Or alternatively from [NPM][npm-url]:
-```shell
-npm install vui-typography
-```
+## Usage
 
-Depending on which installation method you choose, use that path when doing the SASS import:
+Include the [webcomponents.js](http://webcomponents.org/polyfills/) "lite" polyfill (for browsers who don't natively support web components), import `typography.html`, and include the `vui-typography` style:
 
-```scss
-@import 'bower_components/vui-typography/typography.scss';
-// or...
-@import "node_modules/vui-typography/typography.scss";
+```html
+<head>
+	<script src="https://s.brightspace.com/lib/webcomponentsjs/0.7.21/webcomponents-lite.min.js"></script>
+	<link rel="import" href="../vui-typography/typography.html">
+	<style is="custom-style" include="vui-typography"></style>
+</head>
 ```
 
-## Usage
-
 ### Body
 
```

Comment:
Hmmm - this would only be valuable outside the context of the LMS where the base font-size won't be set for them.  We would not want a FRA to hard-code `html { font-size: 20px; }`.  Thoughts?